src/lib/capabilities.tsx

```diff
@@ -121,9 +121,7 @@ export const getCapabilities = (
     restoreVersion: isOwner() || isEditor() || isWriter(),
     /* file handling */
     downloadFiles: true,
-    changeDesignation:
-      (isOwner() || isEditor() || isWriter()) &&
-      allowed(Actions.updateAttachment),
+    moveFile: isOwner() || isEditor() || isWriter() || isAnnotator(),
```

Comment:
I don't think annotators could make changes to the model.

Reply:
updated